packages/utils/ast-utils.js

```diff
@@ -436,6 +436,7 @@ function addProperty(j, p, key, value, action) {
 		});
 		valForNode = objectExp;
 	} else {
+		value = `'${value}'`;
```

Comment:
Some of the code like this is a part of PR #499. I'll rebase and cleanup once it gets merged.

Reply:
do it in this PR. Your code is obviously wrong when you see that you've changed snapshots signature.

Reply:
If we want to handle this in add generator, it's easy as we can do at when we generate/assign literal string but if we think about remove generator which is essentially copying the whole config then it won't be possible to find and reassign every string literal in the webpack config object we read from user file.

Reply:
Move this to generator, AST should not be concerned with this.